package.nls.json

```diff
@@ -86,6 +86,9 @@
 	"githubPullRequests.createDefaultBaseBranch.repositoryDefault": "The default branch of the repository",
 	"githubPullRequests.createDefaultBaseBranch.createdFromBranch": "The branch that the current branch was created from, if known",
 	"githubPullRequests.createDefaultBaseBranch.auto": "When the current repository is a fork, this will work like \"repositoryDefault\". Otherwise, it will work like \"createdFromBranch\".",
+	"githubPullRequests.experimental.chat.description": "Enables the `@githubpr` Copilot chat provider in the chat view. `@githubpr` can help search for issues and pull requests, suggest fixes for issues, and summarize issues, pull requests, and notifications.",
```

Comment:
Is it chat provider or chat participant?

Reply:
Thanks for catching!